### PR TITLE
[Doctest] Add `configuration_bigbird_pegasus.py` and `configuration_big_bird.py`

### PR DESCRIPTION
**Pull request by xlahuerta**
Add `configuration_bigbird_pegasus.py` and `configuration_big_bird.py` to `utils/documentation_tests.txt` for doctest.

Based on issue https://github.com/huggingface/transformers/issues/19487

@ydshieh could you please check it?
Thank you :)

### CODE REVIEW

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 1]
```diff
@@ -94,13 +94,13 @@ class BigBirdConfig(PretrainedConfig):
...
 
-        >>> from transformers import BigBirdModel, BigBirdConfig
+        >>> from transformers import BigBirdConfig, BigBirdModel
 
         >>> # Initializing a BigBird google/bigbird-roberta-base style configuration >>> configuration =
         BigBirdConfig()
 
-        >>> # Initializing a model from the google/bigbird-roberta-base style configuration >>> model =
-        BigBirdModel(configuration)
+        >>> # Initializing a model (with random weights) from the google/bigbird-roberta-base style configuration >>>
```
Comment (ydshieh) [inline src/transformers/models/big_bird/configuration_big_bird.py:102]: Hi, could you fix a bug. The line `>>> model =` should be in a new line. Same for configuration, and same for `BigBirdPegasusConfig`. Thank you :-)

Comment (ydshieh) [reply src/transformers/models/big_bird/configuration_big_bird.py:102]: This bug is on us, not from your change :-)

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 2]
```diff
@@ -91,18 +91,20 @@ class BigBirdConfig(PretrainedConfig):
     Example:
 
     ```python
+        >>> from transformers import BigBirdConfig, BigBirdModel
 
-    ```
+        >>> # Initializing a BigBird google/bigbird-roberta-base style configuration
 
-        >>> from transformers import BigBirdConfig, BigBirdModel
+        >>> configuration = BigBirdConfig()
```
Comment (ydshieh) [inline src/transformers/models/big_bird/configuration_big_bird.py:98]: no need the above new empty line

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 3]
```diff
@@ -91,18 +91,20 @@ class BigBirdConfig(PretrainedConfig):
...
 
-        >>> # Initializing a BigBird google/bigbird-roberta-base style configuration >>> configuration =
-        BigBirdConfig()
+        >>> # Initializing a model (with random weights) from the google/bigbird-roberta-base style configuration
 
-        >>> # Initializing a model (with random weights) from the google/bigbird-roberta-base style configuration >>>
-        model = BigBirdModel(configuration)
+        >>> model = BigBirdModel(configuration)
```
Comment (ydshieh) [inline src/transformers/models/big_bird/configuration_big_bird.py:102]: same

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 4]
```diff
@@ -91,18 +91,20 @@ class BigBirdConfig(PretrainedConfig):
...
 
-        >>> # Accessing the model configuration >>> configuration = model.config
+        >>> # Accessing the model configuration
+
+        >>> configuration = model.config
```
Comment (ydshieh) [inline src/transformers/models/big_bird/configuration_big_bird.py:106]: same

File: src/transformers/models/big_bird/configuration_big_bird.py [hunk 5]
```diff
@@ -91,18 +91,17 @@ class BigBirdConfig(PretrainedConfig):
...
+        >>> # Initializing a model (with random weights) from the google/bigbird-roberta-base style configuration
+        >>> model = BigBirdModel(configuration)
 
-        >>> # Initializing a model from the google/bigbird-roberta-base style configuration >>> model =
-        BigBirdModel(configuration)
-
-        >>> # Accessing the model configuration >>> configuration = model.config
+        >>> # Accessing the model configuration
+        >>> configuration = model.config
+    ```
```
Comment (ydshieh) [inline src/transformers/models/big_bird/configuration_big_bird.py:104]: Hi, after a close look, it seems this config docstring had some more issues previously.

Thank your for fixing this here, but let's put this line and the next line to a single line **```"""** to follow other config files :-)

Also, inside the code examples (i.e. the part with `>>>`), let's unindent those line 🙏 

Sorry for so many unexpected changes here

Comment (ydshieh) [reply src/transformers/models/big_bird/configuration_big_bird.py:104]: And same for BigBirdPegasusConfig 🙏

**Review by ydshieh — APPROVED** [review]
Very nice and clean, thank you for your patience and work @Xabilahu ❤️

**Comment by Xabilahu** [comment]
Thanks for the amazing work you guys do <3

**Pull request merged**